undefined_syms.txt

```diff
@@ -457,3 +457,11 @@ D_0E000000 = 0x0E000000;
 // segment 0x0F
 
 D_0F000000 = 0x0F000000;
+
+ENTRY(entrypoint);
+
+ASSERT ((_bootSegmentRomEnd <= 0x101000), "Error: _bootSegmentRomEnd is larger than 1 MB");
+
+// TODO: This should be checking for an address a lot earlier than this one.
+ASSERT ((_system_heapSegmentStart < 0x80780000), "Error: The game is overflowing the RAM limit for non-overlay segments.");
+ASSERT ((_framebuffer_hiSegmentStart == 0x80780000), "Error: framebuffer_hi shifted?");
```

Comment:
Instead of putting this in `undefined_syms.txt`, could we perhpas create a linker_scripts folder and add a new linker script file (not quite sure what to call it though)? I would also say move `undefined_syms.txt` to this new folder and rename it to `undefined_syms.ld`.

Reply:
I named the new file `linker_scripts/linker_script_extra.ld`. I'm happy to hear a better name